lib/publiccloud/instance.pm

```diff
@@ -379,12 +380,11 @@ sub softreboot
 
     my $duration;
 
-    # On TUNNELED test runs, ensure we are not on the publiccloud instance
     my $prev_console = current_console();
-    # We only need to re-establish the ssh tunnel, if we are in a TUNNELED test run, and if the tunnel is already initialized
-    my $tunneled = get_var('TUNNELED', 0) && get_var("_SSH_TUNNELS_INITIALIZED", 0);
+    # On TUNNELED test runs, we need to re-establish the tunnel
+    my $tunneled = get_var('TUNNELED', 0) && check_var("_SSH_TUNNELS_INITIALIZED", 1);
```

Comment:
does not make a big difference , just asking out of curiosity so don't take this as request to change something : 
why you replace only one get_var with check_var ? for me they have exactly same reasons to stay as get_var and exactly same reasons to be replaced with check_var ?

Reply:
This probably happened while development, no particular reason and I'll change it back.